torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -1999,13 +1999,6 @@ at::Tensor& AtenXlaType::mv_out(at::Tensor& out, const at::Tensor& self,
   return out;
 }
 
-at::Tensor AtenXlaType::narrow(const at::Tensor& self, int64_t dim,
-                               int64_t start, int64_t length) {
-  XLA_FN_COUNTER("xla::");
-  return bridge::AtenFromXlaTensor(
-      XLATensor::narrow(bridge::GetXlaTensor(self), dim, start, length));
```

Comment:
The `XLATensor::narrow()` still have uses?

Reply:
Yup it's used by Select. :D

Reply:
https://github.com/pytorch/xla/blob/de07b24d10b6c53de8e603c5756e4dda6d42cb35/torch_xla/csrc/tensor_ops.cpp#L176

Reply:
👻